index.bs

```diff
@@ -186,9 +186,9 @@ The [=AV1 Image File Format=] defines multiple profiles, which restrict the allo
 The profiles defined in this specification follow the conventions of the [[!MIAF]] specification.
 Images encoded with [[!AV1]] and not meeting the restrictions of the defined profiles may still be compliant to this [=AV1 Image File Format=] if they adhere to the general AVIF requirements.
 
-[=AV1 Image File Format=] supports High Dynamic Range (HDR) and Wide Color Gamut (WCG) images as well as Standard Dynamic Range (SDR). It supports monochrome images as well as multi-channel images with all the bit depths and color spaces specified in [[!AV1]].
+The [=AV1 Image File Format=] supports High Dynamic Range (HDR) and Wide Color Gamut (WCG) images as well as Standard Dynamic Range (SDR). It supports monochrome images as well as multi-channel images with all the bit depths and color spaces specified in [[!AV1]], and other bit depths with [=Sample Transform Derived Image Items=]. The [=AV1 Image File Format=] also supports transparency (alpha) and other types of data such as depth maps through auxiliary [=AV1 bitstreams=].
```

Comment:
Leo, Yannis: Just curious -- are gain maps also auxiliary image items, or are they inputs to derived image items?

I don't really know the purpose of this paragraph, so I am not sure if it should be comprehensive. Do you think it should mention using image grid derived image items to exceed the maximum AV1 frame size (of 64K x 64K)?

Reply:
> are gain maps also auxiliary image items, or are they inputs to derived image items?

They are inputs to derived items (at least as far as `tmap` goes).

Prior to 21496-1, the way Apple did gain maps was as a proprietary auxiliary image.